Classes/ScrnLAW.uc

```diff
@@ -1,9 +1,32 @@
 class ScrnLAW extends LAW;
 
+//overwriting to increase raise animation play rate
+simulated function ZoomIn(bool bAnimateTransition)
+{
+    if( Level.TimeSeconds < FireMode[0].NextFireTime )
+    {
+        return;
+    }
+    super.ZoomIn(bAnimateTransition);
+
+    if( bAnimateTransition )
+    {
+        if( bZoomOutInterrupted )
+        {
+            PlayAnim('Raise',2.7,0.1); //increased to 2.7
```

Comment:
Let's put "2.7" into a variable `RaiseAnimRate`, to allow child classes to easily override it if needed.